### ISSUE TITLE
Tag support for aws_cognito_identity_pool

### ISSUE THREAD

**Issue by jfharden** [terraform-providers/terraform-provider-aws#9312]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Please could you add tags as an attribute for aws_cognito_identity_pool. The amazon docs say tagging is supported via the API https://docs.aws.amazon.com/cognito/latest/developerguide/tagging.html

### New or Affected Resource(s)

* aws_cognito_identity_pool

### Potential Terraform Configuration

```hcl
resource "aws_cognito_identity_pool" "developers" {
  identity_pool_name               = "DeveloperPool"
  allow_unauthenticated_identities = false
  developer_provider_name          = "DeveloperProvider"

  tags = {
    CostCenter = "Engineering" 
  }
}
```

### References

* https://docs.aws.amazon.com/cognito/latest/developerguide/tagging.html

**Comment by ewbankkit** [comment]
Similar:
* https://github.com/terraform-providers/terraform-provider-aws/issues/6301

**Comment by bflad** [comment]
Support for tagging `aws_cognito_identity_pool` resources has been merged and will release with version 2.23.0 of the Terraform AWS Provider, in the next day or two. Thanks to @Alexanderlakovlev for the implementation. 👍
